tools/cross-compile.sh

```diff
@@ -106,6 +106,10 @@ echo "Windows 64"
 GOOS=windows GOARCH=amd64 make build
 create_artefact_windows "win64"
 
+echo "Windows ARM64"
```

Comment:
```suggestion
echo "Windows ARM 64"
```

Just to have it similar to the existing `"Linux ARM 64"` :wink:

Reply:
Done